services/QuillLMS/app/controllers/students_controller.rb

```diff
@@ -113,12 +118,12 @@ def join_classroom
     classroom_id = params["classroom"]
     unit_id = params["unit_id"]
 
-    return unless classroom_id && unit_id
-
     classroom_unit = ClassroomUnit.find_by(classroom_id: classroom_id, unit_id: unit_id)
 
     unless classroom_unit && classroom_unit.assigned_student_ids.include?(current_user.id)
-      flash[:error] = 'Sorry, you do not have access to this activity pack because it has not been assigned to you. Please contact your teacher.'
+      flash[:error] = t('activity_link.errors.activity_pack_not_assigned')
```

Comment:
I do like the move to translation strings in these situations.  Just figured I'd cll that out since we should probably do more of it.